src/query/api/v1/handler/prometheus/common.go

```diff
@@ -304,6 +304,35 @@ func renderDefaultTagCompletionResultsJSON(
 	return jw.Close()
 }
 
+// RenderListTagResultsJSON renders list tag results to json format.
+func RenderListTagResultsJSON(
```

Comment:
Also, is there a scenario where `status` isn't `success`?

Reply:
No; if it errors, it won't use this render function but rather the default error printer

Reply:
What's the point of having a status field then lol

Reply:
It's expected in the Prom output unfortunately